scout/parse/variant/genotype.py

```diff
@@ -318,49 +322,48 @@ def get_read_depth(variant, pos, alt_depth, ref_depth):
 
 
 def get_ref_depth(
-    variant,
-    pos,
-    paired_end_ref,
-    split_read_ref,
-    spanning_ref,
-    flanking_ref,
-    inrepeat_ref,
-    spanning_mei_ref,
-):
+    variant: cyvcf2.Variant,
+    pos: int,
+    paired_end_ref: int,
+    split_read_ref: int,
+    spanning_ref: int,
+    flanking_ref: int,
+    inrepeat_ref: int,
+    sd_ref: int,
+    spanning_mei_ref: int,
+) -> int:
     """Get reference read depth"""
     ref_depth = int(variant.gt_ref_depths[pos])
     if ref_depth != -1:
         return ref_depth
 
     REF_ITEMS_LIST: List[tuple] = [
+        (sd_ref,),
         (paired_end_ref, split_read_ref),
         (spanning_ref, flanking_ref, inrepeat_ref),
     ]
 
-    for list in REF_ITEMS_LIST:
-        if all(item is None for item in list):
-            continue
-        ref_depth = 0
-        for item in list:
-            if item:
-                ref_depth += item
+    for items in REF_ITEMS_LIST:
+        if any(item is not None for item in items):
+            ref_depth = sum(item for item in items if item)
```

Comment:
```suggestion
            ref_depth = sum(item for item in items if item is not None)
```

 I think, so you get also the items that are 0?

Reply:
Yes, I didn't really think about it, but we are summing, not counting so either works. 😊